### PR TITLE
Added /call url to allow for calls to other clients or phone numbers

### PR DESCRIPTION
**Pull request by tmconnors**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 IDENTITY = 'voice_test'
 CALLER_ID = 'quick_start'
+CALLER_PHONE = '+1****'
```
Comment (idelgado) [inline server.py]: Should we default to a real US number here instead?

Comment (tmconnors) [inline server.py]: Yeah. The quickstart guide will need a section about obtaining a twilio number and replacing this value.

Comment (idelgado) [inline server.py]: Cool. This is fine for now.

Comment (paynerc) [inline server.py]: We should also specify this as an environment variable like we do with the other values here so we can have the Heroku deployment still work easily.

**Pull request closed without merging**